data/autoyast_sle15/autoyast_rt_textmode.xml

```diff
@@ -1,163 +1,163 @@
+        <product>SLE_RT_TEST_OPENQA</product>
+        <product_dir>/</product_dir>
+        <priority config:type="integer">86</priority>
+        <name>Latest RT packages from IBS2</name>
+        <confirm_license config:type="boolean">false</confirm_license>
+        <ask_on_error config:type="boolean">false</ask_on_error>
+      </listentry>
+    </add_on_products>
+  </add-on>
+  <!-- END OF HACK!!!-->
+  <general>
+    <mode>
+      <confirm config:type="boolean">false</confirm>
+    </mode>
+  </general>
+  <bootloader>
+    <global>
+      <timeout config:type="integer">-1</timeout>
+      <append>ignore_loglevel</append>
+    </global>
+  </bootloader>
+  <report>
+    <errors>
+      <log config:type="boolean">true</log>
+      <show config:type="boolean">true</show>
+      <timeout config:type="integer">0</timeout>
+    </errors>
+    <messages>
+      <log config:type="boolean">true</log>
+      <show config:type="boolean">true</show>
+      <timeout config:type="integer">0</timeout>
+    </messages>
+    <warnings>
+      <log config:type="boolean">true</log>
+      <show config:type="boolean">true</show>
+      <timeout config:type="integer">0</timeout>
+    </warnings>
+    <yesno_messages>
+      <log config:type="boolean">true</log>
+      <show config:type="boolean">true</show>
+      <timeout config:type="integer">0</timeout>
+    </yesno_messages>
+  </report>
+  <software>
+    <products config:type="list">
+    <!-- HACK!!! -->
+        <!--product>SLE_RT</product-->
+      <product>SLES</product>
+    <!-- END OF HACK!!!-->
+    </products>
+    <install_recommended config:type="boolean">true</install_recommended>
+    <!--kernel>kernel-rt</kernel-->
+    <packages config:type="list">
+      <package>sle-module-server-applications-release</package>
+      <package>sle-module-development-tools-release</package>
+      <package>sle-module-basesystem-release</package>
+    </packages>
+    <patterns config:type="list">
+      <pattern>apparmor</pattern>
+      <pattern>base</pattern>
+      <pattern>basesystem</pattern>
+      <!--pattern>documentation</pattern-->
+      <!--pattern>enhanced_base</pattern-->
+      <pattern>minimal_base</pattern>
+      <!-- HACK!!!-->
+      <pattern>rt_sles</pattern>
+      <!-- END OF HACK!!!-->
+    </patterns>
+  </software>
+  <networking>
+    <keep_install_network config:type="boolean">true</keep_install_network>
+  </networking>
+  <users config:type="list">
+    <user>
+      <encrypted config:type="boolean">true</encrypted>
+      <fullname>Bernhard M. Wiedemann</fullname>
+      <gid>100</gid>
+      <home>/home/bernhard</home>
+      <password_settings>
+        <inact>-1</inact>
+        <max>99999</max>
+        <min>0</min>
+        <warn>7</warn>
+      </password_settings>
+      <shell>/bin/bash</shell>
+      <uid>1000</uid>
+      <user_password>$6$vYbbuJ9WMriFxGHY$gQ7shLw9ZBsRcPgo6/8KmfDvQ/lCqxW8/WnMoLCoWGdHO6Touush1nhegYfdBbXRpsQuy/FTZZeg7gQL50IbA</user_password>
```

Comment:
maybe this one could be fixed too

Reply:
i could test it but i think this is correct. the only problem was that `encrypted` was set false.